crates/hir/src/lib.rs

```diff
@@ -459,6 +450,20 @@ impl ModuleDef {
         acc
     }
 
+    pub fn as_trait(self) -> Option<Trait> {
```

Comment:
Generally we've avoided adding those methods, they can have a lot of boilerplate and impact on compile time. I'd prefer if they won't exist.